### ISSUE TITLE
Keep binary files out of repository and in Releases tab

### ISSUE THREAD

**Issue by wqweto** [tannerhelland/VBIDEUtils#2]
Usually compiled files are uploaded under [Releases tab](https://github.com/tannerhelland/VBIDEUtils/releases) along with a zip of the sources from which these are compiled out. This way only "real" source files (for main project, tests, samples) are kept under version control in the repository.

In VB6 case probably compatibility targets must be kept in the repository too for the source to be able to be opened in VBIDE, probably these will need versioning too.

I tried to use `.tlb` files for compatibility but these can be used only for _Project_ compatibility targets. Binary compatibility needs a compiled VB6 binary as target.

Just my $.02

**Comment by tannerhelland** [comment]
Yes, I completely agree.  I actually had the binary files in a /Bin folder but changed my mind at the last minute.

From what I saw (sorry, not much time to investigate better), Thierry set up the VBG so that the compiled files are required in the base folder?  Without them registered from that location, various projects from the repo will fail to open?  Maybe I didn't look closely enough.  Anyway, that understanding is why I changed my mind; given the technical proficiency of many vbforums users, I thought it might be helpful to leave the binaries there to reduce confusion.  

Of course this is a debacle for version control, but I don't know how much the files will be updated regularly.  I myself don't have time to really work on this - I just want to move as many VB sources to GitHub as possible, before owners disappear or forget about their code.

Anyway, in the coming days I can remove the binaries.  I am open to any suggestions for improving the "plug-and-play" nature of the add-in, since it is still in rough shape for the average user, but it is hard to find a balance when compiling from source can be "complicated" for many VB guys.
